app/src/main/java/net/squanchy/tweets/view/TweetUrlSpan.kt

```diff
@@ -0,0 +1,39 @@
+package net.squanchy.tweets.view
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.provider.Browser
+import android.text.TextPaint
+import android.text.style.ClickableSpan
+import android.view.View
+
+import timber.log.Timber
+
+class TweetUrlSpan internal constructor(
```

Comment:
I think the whole class could be internal

Reply:
👍